### PR TITLE
Show entries on Port Forwarding screenshot

### PR DESCRIPTION
**Pull request by s0nea**
Fixes: https://github.com/rancher-sandbox/rancher-desktop/issues/5753

Note:
- the PR is based on https://github.com/rancher-sandbox/rancher-desktop/pull/5752 and needs to be merged afterwards.
- ~I tested the PR on Linux and macOS. It would be great if someone could give it a try on Windows.~

### CODE REVIEW

File: pkg/rancher-desktop/backend/kube/lima.ts [hunk 1]
```diff
@@ -186,7 +187,7 @@ export default class LimaKubernetesBackend extends events.EventEmitter implement
           return k3sConfigString;
         }));
 
-    this.client = new KubeClient();
+    this.client = !process.env.RD_MOCK_FOR_SCREENSHOTS ? new KubeClient() : new KubeClientMock();
```
Comment (rak-phillip) [inline pkg/rancher-desktop/backend/kube/lima.ts:190]: I can't say that I'm very keen on code dedicated to mocking screenshots making it into `lima.ts` and `wsl.ts`. Have we investigated other patterns that we might be able to apply that will achieve something similar?

Comment (rak-phillip) [reply pkg/rancher-desktop/backend/kube/lima.ts:190]: Would we be able to utilize the mock backend that we specify in the factory instead?

https://github.com/rancher-sandbox/rancher-desktop/blob/7e32cb5fa5f99f5a318dbd51c1edb131cb0ae7bd/pkg/rancher-desktop/backend/factory.ts#L15-L17

Comment (s0nea) [reply pkg/rancher-desktop/backend/kube/lima.ts]: I pushed an update to the PR. I think it's a little bit better than before. Although I don't think it's _perfect_. Would you mind taking a look at it again?

**Review by ericpromislow — APPROVED** [review]
Works on all three platforms.

**Pull request merged**